.github/workflows/dart.yml

```diff
@@ -90,9 +90,10 @@ jobs:
       - name: Run builders
         run: |
           multipack --only $PACKAGE pub run build_runner build --delete-conflicting-outputs
-      - name: Check build diff
-        run: |
-          multipack --only $PACKAGE exec git diff --exit-code **/*.gql.dart
+      # Broken for no discernable reason
```

Comment:
This removes way too many useful regression tests.

A better solution could be to sort literals to guarantee stable order.

See https://github.com/gql-dart/gql/pull/137